core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -9585,4 +9585,46 @@ private void checkJoinAssociateRuleWithTopAlwaysTrueCondition(boolean allowAlway
         .withRule(CoreRules.MULTI_JOIN_OPTIMIZE)
         .check();
   }
+
+  @Test void testIntersectToExistsRuleOneField() {
```

Comment:
since plans are hard to read, I would encourage you to also add some quidem tests, these are much simpler to read and write

Reply:
> since plans are hard to read, I would encourage you to also add some quidem tests, these are much simpler to read and write

@mihaibudiu Could you give me a description of quidem tests? I don't quite understand how to write them now.

Reply:
Look at the files with iq suffix

Reply:
> Look at the files with iq suffix

@mihaibudiu Thanks, I have seen the file with this suffix. Do I need to write a SQL test to output the final result through the optimization rules? Is there a similar example for reference?

Reply:
Just as @mihaibudiu  said, currently this quidem file cannot specify specific optimizations. Before Quidem's test, we wrote such test cases in JdbcTest. Please feel free to check it.